showcases/patternhub/pages/foundations/colors/color-modes.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import DefaultPage from '../../../components/default-page';
+import { DBButton, DBTag, DBCard } from '../../../../../output/react/src';
+import { SEMANTICS } from '../../../../../packages/components/src/shared/constants';
+
+const colors = ['neutral', ...SEMANTICS];
+
+const ColorOverview = () => {
+	const [colorScheme, setColorScheme] = useState<string>('light');
+	const [colorScheme2, setColorScheme2] = useState<string>('light');
+	return (
+		<DefaultPage>
+			<h1>Color Schemes</h1>
+			<p>
+				You can use <code>data-mode="light/dark"</code> to force a
```

Comment:
```suggestion
				You can use <code>data-mode="light|dark"</code> to force a
```

At other places, you're using "|" instead of "/" to separate possible values.